express-zod-api/src/zod-plugin.ts

```diff
@@ -68,7 +69,7 @@ declare module "zod/v4" {
       mapping: U,
     ): z.ZodPipe<
       z.ZodPipe<this, z.ZodTransform>, // internal type simplified
-      z.ZodObject<Remap<Shape, U, V> & Intact<Shape, U>, Config>
+      z.ZodObject<R.Remap<Shape, U> & Intact<Shape, U>, Config>
```

Comment:
check if it's actually exposed/resolved into the build

Reply:
it's not. but there is an option in `tsup`: `dts.resolve`, however, it's not documented how exactly it works (it does not)